editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -531,6 +534,104 @@ describe('actions', () => {
         expect(got.selection).toEqual(tt.wantSelection)
       })
     })
+
+    it('can delete render props from an element in a map expression', async () => {
+      const editor = await renderTestEditorWithModel(
+        createModifiedProject({
+          [StoryboardFilePath]: `import * as React from 'react'
+        import * as Utopia from 'utopia-api'
+        import { Storyboard, Scene } from 'utopia-api'
+        
+        export function Card({ header, children }) {
+          return (
+            <div data-uid='dbc'>
+              <h2 data-uid='4b3'>{header}</h2>
+              {children}
+            </div>
+          )
+        }
+        
+        export var storyboard = (
+          <Storyboard data-uid='sb'>
+            <Scene
+              style={{
+                width: 521,
+                height: 266,
+                position: 'absolute',
+                left: 717,
+                top: 597,
+                backgroundColor: 'white',
+              }}
+              data-uid='scene'
+              data-testid='scene'
+              commentId='sce'
+            >
+              {
+                // @utopia/uid=map
+                ["test"].map(() => (
+                  <Card
+                    data-uid='card'
+                    header={<h3 data-uid='b82'>woot</h3>}
```

Comment:
instead of b82, please give this a more noticeable uid, like render-prop-to-be-deleted :)

Reply:
added a different uid in https://github.com/concrete-utopia/utopia/pull/5260/commits/ef26dccd82d9d08ccbbd2986ed8f1f88f4aa13d1